pkg/oc/cli/cmd/debug.go

```diff
@@ -129,7 +129,7 @@ func NewCmdDebug(fullName string, f *clientcmd.Factory, in io.Reader, out, errou
 	}
 
 	cmd := &cobra.Command{
-		Use:     "debug RESOURCE/NAME [ENV1=VAL1 ...] [-c CONTAINER] [options] [-- COMMAND]",
+		Use:     "debug RESOURCE/NAME [ENV1=VAL1 ...] [-c CONTAINER] [flags] [-- COMMAND]",
```

Comment:
Not that I object, but why do I care about this change?

Reply:
Because cobra changed to auto append `[flags]` if you don't include it in your usage, which screws up usage for things like rsh